Find a functionally equivalent code:android.widget.CalendarView.setDate:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the selected date in milliseconds since January 1, 1970 00:00:00 in
 * {@link TimeZone#getDefault()} time zone.
 *
 * @param date The selected date.
 *
 * @throws IllegalArgumentException of the provided date is before the
 * minimal or after the maximal date.
 *
 * @see #setDate(long, boolean, boolean)
 * @see #setMinDate(long)
 * @see #setMaxDate(long)
 */

@@ -1,3 +1,3 @@
 {
-    setDate(date, false, false);
+    mDelegate.setDate(date);
 }

